markdown/developers/modules.md

```diff
@@ -376,15 +378,45 @@ All non-mandatory command-line tool non-file arguments MUST be provided as a str
       }
       ```
 
-2. Software that can be piped together SHOULD be added to separate module files
+4. Software that can be piped together SHOULD be added to separate module files
    unless there is a run-time, storage advantage in implementing in this way. For example,
    using a combination of `bwa` and `samtools` to output a BAM file instead of a SAM file:
 
    ```bash
-   bwa mem | samtools view -B -T ref.fasta
+   bwa mem $args | samtools view $args2 -B -T ref.fasta
+   ```
+
+5. Each tool in a multi-tool module MUST have an `$args` e.g.,
+
+   ```bash
+   bwa mem $args | samtools view $args2 -B -T ref.fasta | samtools sort $args3
    ```
 
-3. Where applicable, the usage and generation of compressed files SHOULD be enforced as input and output, respectively:
+      or
+
+   ```bash
+   <tool> \\
+      <subcommand> \\
+      $args
+
+   gzip \\
+       $args2
+   ```
+
+   The numbering of each `$args` MUST correspond to the order of the tools, and MUST be documented in `meta.yml`.
```

Comment:
Documenting args in meta.yml is a bit of a departure, right? Are there examples?

Reply:
A little, but I think it's important that people don't forget this as it will not be in the modules template yet (and see the changes below).

Examples are coming soon, @mashehu and I just decided on this right now so it's fresh

Reply:
Yes, just numbering. OK I'll give a more explicit example here too.